### PR TITLE
1-Update pricing.mdx

### PR DESCRIPTION
**Pull request by Michellehbn**
endpoint pricing and instance updates

### CODE REVIEW

File: docs/source/pricing.mdx [hunk 1]
```diff
@@ -23,32 +23,41 @@ You can find the hourly pricing for all available instances for 🤗 Inference E
...
+| Provider | Instance Size | Hourly rate | GPUs | Memory | Architecture | ID                      |
+| -------- | ------------- | ----------- | ---- | ------ | ------------ | ----------------------- |
+| aws      | small         | $0.500      | 1    | 14GB   | NVIDIA T4    | aws-g4dn-xlarge-small   |
+| aws      | medium        | $1.000      | 1    | 24GB   | NVIDIA A10G  | aws-g5-2xlarge-medium   |
+| aws      | large         | $3.000      | 4    | 56GB   | NVIDIA T4    | aws-g4dn-12xlarge-large |
+| aws      | xlarge        | $4.000      | 1    | 80GB   | NVIDIA A100  | aws-p4de-xlarge         |
+| aws      | xxlarge       | $5.000      | 4    | 96GB   | NVIDIA A10G  | aws-g5-12xlarge-xxlarge |
+| aws      | 2xlarge       | $8.000      | 2    | 160GB  | NVIDIA A100  | aws-p4de-2xlarge        |
+| aws      | 4xlarge       | $16.000     | 4    | 320GB  | NVIDIA A100  | aws-p4de-4xlarge        |
+| aws      | 8xlarge       | $32.000     | 8    | 640GB  | NVIDIA A100  | not available           |
```
Comment (philschmid) [inline docs/source/pricing.mdx:54]:
```suggestion
| aws      | 8xlarge       | $32.000     | 8    | 640GB  | NVIDIA A100  | aws-p4de-8xlarge           |
```
is it really not available? or just requires a reserved on?

File: docs/source/pricing.mdx [hunk 2]
```diff
@@ -23,32 +23,41 @@ You can find the hourly pricing for all available instances for 🤗 Inference E
...
+| gcp      | x1            | $1.000      | 1    | 24GB   | NVIDIA L4    | gcp-nvidia-l4-x1        |
```
Comment (philschmid) [inline docs/source/pricing.mdx:55]:
```suggestion
| gcp      | x1            | $0.500      | 1    | 24GB   | NVIDIA T4    | gcp-nvidia-t4-x1        |
| gcp      | x1            | $1.000      | 1    | 24GB   | NVIDIA L4    | gcp-nvidia-l4-x1        |
```
adds T4

**Review by philschmid — APPROVED** [review]
Thank you for updating! 🤗 Added 2 comments.

File: docs/source/pricing.mdx (hunk 1 above)
Comment (jeffboudier) [inline docs/source/pricing.mdx:54]:
```suggestion
| aws      | small         | $0.5      | 1    | 14GB   | NVIDIA T4    | aws-g4dn-xlarge-small   |
| aws      | medium        | $1      | 1    | 24GB   | NVIDIA A10G  | aws-g5-2xlarge-medium   |
| aws      | large         | $3      | 4    | 56GB   | NVIDIA T4    | aws-g4dn-12xlarge-large |
| aws      | xlarge        | $4      | 1    | 80GB   | NVIDIA A100  | aws-p4de-xlarge         |
| aws      | xxlarge       | $5      | 4    | 96GB   | NVIDIA A10G  | aws-g5-12xlarge-xxlarge |
| aws      | 2xlarge       | $8      | 2    | 160GB  | NVIDIA A100  | aws-p4de-2xlarge        |
| aws      | 4xlarge       | $16     | 4    | 320GB  | NVIDIA A100  | aws-p4de-4xlarge        |
| aws      | 8xlarge       | $32     | 8    | 640GB  | NVIDIA A100  | not available           |
```

File: docs/source/pricing.mdx (hunk 2 above)
Comment (jeffboudier) [inline docs/source/pricing.mdx:55]:
```suggestion
| gcp      | x1            | $1      | 1    | 24GB   | NVIDIA L4    | gcp-nvidia-l4-x1        |
```

File: docs/source/pricing.mdx [hunk 3]
```diff
@@ -23,32 +23,41 @@ You can find the hourly pricing for all available instances for 🤗 Inference E
...
+| gcp      | x4            | $5.000      | 4    | 96GB   | NVIDIA L4    | gcp-nvidia-l4-x4        |
+| gcp      | x1            | $6.000      | 1    | 80 GB  | NVIDIA A100  | gcp-nvidia-a100-x1      |
+| gcp      | x2            | $12.000     | 2    | 160 GB | NVIDIA A100  | gcp-nvidia-a100-x2      |
+| gcp      | x4            | $24.000     | 4    | 320 GB | NVIDIA A100  | gcp-nvidia-a100-x4      |
+| gcp      | x8            | $48.000     | 8    | 640 GB | NVIDIA A100  | gcp-nvidia-a100-x8      |
```
Comment (jeffboudier) [inline docs/source/pricing.mdx:60]:
```suggestion
| gcp      | x4            | $5      | 4    | 96GB   | NVIDIA L4    | gcp-nvidia-l4-x4        |
| gcp      | x1            | $6      | 1    | 80 GB  | NVIDIA A100  | gcp-nvidia-a100-x1      |
| gcp      | x2            | $12     | 2    | 160 GB | NVIDIA A100  | gcp-nvidia-a100-x2      |
| gcp      | x4            | $24     | 4    | 320 GB | NVIDIA A100  | gcp-nvidia-a100-x4      |
| gcp      | x8            | $48     | 8    | 640 GB | NVIDIA A100  | gcp-nvidia-a100-x8      |
```

**Review by jeffboudier — APPROVED** [review]
Tried to simplify pricing removing the trailing zeros.
Should we update /pricing in the same PR?
Is the reason we keep [small,medium,large,xlarge] nomenclature as we introduce [x1,x2,x3,x4] (better IMO) because of backward compatibility?

File: docs/source/pricing.mdx (hunk 1 above)
Comment (Michellehbn) [reply docs/source/pricing.mdx:54]: I'll check again

**Comment by Michellehbn** [comment]
> Tried to simplify pricing removing the trailing zeros. Should we update /pricing in the same PR? Is the reason we keep [small,medium,large,xlarge] nomenclature as we introduce [x1,x2,x3,x4] (better IMO) because of backward compatibility?

Thanks, @jeffboudier ! It looks better as you recommended! Let me check on nomenclature update to x1,etc  for AWS+Azure to make sure its aligned in the backend cc @XciD for viz

File: docs/source/pricing.mdx (hunk 1 above)
Comment (Michellehbn) [reply docs/source/pricing.mdx]: I think it's just a reserved instance, we can look at unlocking this tho later! cc @co42

**Review by philschmid — APPROVED** [review]

**Review by jeffboudier — APPROVED** [review]

**Pull request merged**